packages/wallets/src/features/cfd/modals/DxtradeEnterPasswordModal/DxtradeEnterPasswordModal.tsx

```diff
@@ -233,6 +239,21 @@ const DxtradeEnterPasswordModal = () => {
         sendEmail,
     ]);
 
+    if (status === 'error' && error?.error?.code === 'PasswordReset') {
+        return (
+            <ModalWithBackdropWrapper>
+                <PasswordLimitExceeded
+                    onPrimaryClick={hide}
+                    onSecondaryClick={() => {
+                        sendEmail({
+                            platform: dxtradePlatform,
+                        });
+                    }}
+                />
+            </ModalWithBackdropWrapper>
```

Comment:
This is also fine imo, because I dont think we should continue extending the `ModalProvider` to handle all of these edge cases and adding more and more options to handle this, it makes using `ModalProvider` much more difficult